source/acquaman/AMContinuousScanActionController.cpp

```diff
@@ -277,508 +273,11 @@ void AMContinuousScanActionController::fillDataMaps(AMAgnosticDataAPIDataAvailab
 	Q_UNUSED(message)
 }
 
-bool AMContinuousScanActionController::generateAnalysisMetaInfo()
```

Comment:
In cases where I thought the function might have some applicability in a
general sense I left them in the AM version. I moved the ones that made
specific use of the scaler or Amptek.